src/services/contact.js

```diff
@@ -0,0 +1,29 @@
+import React, { useRef } from 'react';
+import emailjs from '@emailjs/browser';
+
+export const ContactUs = () => {
+    const form = useRef();
+
+    const sendEmail = (e) => {
+        e.preventDefault();
+
+        emailjs.sendForm('service_iwqsu3f', 'template_d0s4lc6', form.current, 'SMrzCAgEF5PcNX7Jo')
+            .then((result) => {
+                console.log(result.text);
+            }, (error) => {
+                console.log(error.text);
```

Comment:
No console logs in the code please. Delete all your console logs after youre done with testing your code. You can use alerts and whatnot for the functionality.